### PR TITLE
Advance config options for PortChannels. #52

### PR DESCRIPTION
**Pull request by rahul-1475**
Issues: 

- the backednd will prompt the error i.e, - Couldn't discover Port Channel on device 10.10.229.58, Reason: invalid literal for int() with base 10: '2-4'
- sometimes the response is just the trunk vlans only

Fixes:

- resolved issue while adding to db using function - format_and_get_trunk_vlans in utils.py 
- added test cases test_port_channel_vlan_members_in_series.
- fixes typo issue for acces_vlan.

### CODE REVIEW

**Comment by rahul-1475** [comment]
orca_nw_lib pr:

- https://github.com/STORDIS/orca_nw_lib/pull/73

File: pyproject.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ djangorestframework = "^3.14.0"
 django-cors-headers = "^3.14.0"
 pytest = "^7.3.2"
 pytest-django = "^4.7.0"
-orca-nw-lib = "^1.3.16"
+orca-nw-lib = "^1.3.17"
```
Comment (kamalkrbh) [inline pyproject.toml:16]: @rahul-1475 , Version need to be updated.

File: network/test/test_port_chnl.py [hunk 2]
```diff
@@ -551,3 +579,113 @@ def test_port_channel_vlan_members(self):
...
+        # cleaning up port channel if it exists
+        self.perform_del_port_chnl({"mgt_ip": device_ip, "lag_name": port_channel})
+
+        # adding port channel
+        self.perform_add_port_chnl(request_body)
+        response = self.get_req("device_port_chnl", request_body)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        members = response.json().get("vlan_members")
+        self.assertEqual(members.get("trunk_vlans"), [vlan_1_id, vlan_2_id, vlan_3_id])
+        self.assertEqual(members.get("access_vlan"), vlan_2_id)
```
Comment (kamalkrbh) [inline network/test/test_port_chnl.py:662]: @rahul-1475 , Assert all other portchannel properties as well.

File: network/test/test_port_chnl.py [hunk 3]
```diff
@@ -551,3 +579,113 @@ def test_port_channel_vlan_members(self):
...
+        self.assertEqual(response.json()["vlanid"], vlan_3_id)
+        self.assertEqual(response.json()["name"], vlan_3_name)
+
+        request_body = {
+            "mgt_ip": device_ip,
+            "lag_name": port_channel,
+            "mtu": 9100,
+            "admin_status": "up",
+            "vlan_members": {
+                "trunk_vlans": [f"{vlan_1_id}..{vlan_3_id}"],
```
Comment (kamalkrbh) [inline network/test/test_port_chnl.py:648]: @rahul-1475 , as along with ".." , "-" is also supported so better to test with both regular expressions.

File: pyproject.toml (hunk 1 above)
Comment (rahul-1475) [reply pyproject.toml]: updated with 1.3.18

File: network/test/test_port_chnl.py [hunk 4]
```diff
@@ -551,3 +579,113 @@ def test_port_channel_vlan_members(self):
...
+                "enabled": False,
+                "description": "Test_Vlan3",
+            },
+        ]
+
+        response = self.put_req(
+            "vlan_config",
+            req_payload,
+        )
+        print("---------", response.content)
```
[the remark this answers is not in the document]
Comment (rahul-1475) [reply network/test/test_port_chnl.py]: Removed print statement

File: network/test/test_port_chnl.py (hunk 2 above)
Comment (rahul-1475) [reply network/test/test_port_chnl.py:784]: asserted other portchannel attributes

File: network/test/test_port_chnl.py (hunk 3 above)
Comment (rahul-1475) [reply network/test/test_port_chnl.py:651]: added new test case to validate "-".

**Pull request merged**